lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>&nbsp;Klasifikátor, který vše, co dostane, označuje&nbsp;za jablko, má tedy na této množině 90% úspěšnost.    (+ 3 čárky)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>